dist/common/sysconfig/scylla-jmx

```diff
@@ -27,3 +27,6 @@ SCYLLA_JMX_LOCAL="-l /opt/scylladb/jmx"
 
 # allow debug
 #SCYLLA_JMX_DEBUG="-d"
+
+# specify JVM options
+SCYLLA_JMX_JVM_OPTS=""
```

Comment:
why not using JVM_OPTS directly and export them?
would save you later the hack with --

Reply:
Seems like it's JAVA_TOOL_OPTIONS not JVM_OPTS, for environment variable.

https://stackoverflow.com/questions/2011311/running-java-with-java-opts-env-variable-has-no-effect
https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html

I'll send new version using the variable.

Reply:
Pushed changes.

Reply:
nice, lgtm from my side, feel free to merge